### PR TITLE
Auth sources

### PR DESCRIPTION
**Pull request by mketchum**
Here are the updates needed to add support for the authentication sources. I ended up creating our own CSS because we didn't quite get full CSS support in the UI library. The SVG backgrounds are pulled from the UI library and encoded for CSS use. They have uploaded the icons to CDN if we want to reference them via URL, but encoded the SVG info means customers won't have to call out to a 3rd party to get the logos.

I'm attempting to add tests but having trouble learning how to mock in jest so those will be coming soon. I just wanted to get the MR out sooner rather than later.

### CODE REVIEW

File: src/partials/username_recovery_email_sent.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="card">
+s<div class="card">
```
[the remark this answers is not in the document]
Comment (mketchum) [reply src/partials/username_recovery_email_sent.hbs]: Ah, good eyes. Definitely didn't mean to change this. At least it wasn't my password :). 

Fixed

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "clean": "rimraf dist test/coverage",
     "lint": "eslint ./src/**.js",
     "coverage": "jest --coverage --config=jest.config.js",
-    "test": "jest --config=jest.config.js",
+    "test": "mocha",
```
Comment (khosravian) [inline package.json:13]: is this necessary?

Comment (mketchum) [reply package.json]: Nope, not even sure how that happened. Probably why webstorm was having so much trouble with tests :). Changed it back

Comment (khosravian) [reply package.json]: 👍

**Pull request merged**